resources/definitions/fdmprinter.def.json

```diff
@@ -1273,8 +1273,8 @@
                 },
                 "optimize_wall_printing_order":
                 {
-                    "label": "Optimize Wall Printing Order",
-                    "description": "Optimize the order in which walls are printed so as to reduce the number of retractions and the distance travelled. Most parts will benefit from this being enabled but some may actually take longer so please compare the print time estimates with and without optimization. First layer is not optimized when choosing brim as build plate adhesion type.",
+                    "label": "Order Inner Walls By Inset",
```

Comment:
Just to be clear, the term inset in this context isn't the same as our regular used inset term in the code.

Reply:
Do you have a proposal for what it can be called?

Reply:
Sorry, this wasn't a change request. Inset is the proper name i.m.o. I just wanted to make sure I was on the same page with you.